### PR TITLE
Html gui refactor

### PR DESCRIPTION
**Pull request by nicknikolov**

### CODE REVIEW

**Comment by vorg** [comment]
1. Comment gui looks broken

screen shot 2018-03-20 at 22 06 51

2. Field labels needs more space. Please test with real graphs like ABB / Orbiter node.

3. I'm not sold on the new slider style. There is more space and therefore precision but it's heavy visually. It might be colors (we need to use neutral grey not green) or use of fill instead of thumb. 

screen shot 2018-03-20 at 21 43 00

4. How much styling should happen here in this pull request or how much we "recover" after merging with master. e.g trash icon is too big? Or padding between rows.

5. I find the style of [code after return](https://github.com/pex-gl/pex-nodes/blob/b6318311818b2548d2ddb9f028c544eefb6f6707/components/graph-editor/inspector/index.js#L195) confusing. It's hard to follow as scope spans multiple pages. Eg. parsePort in inspector.sj:219 uses assetsNames defined in inspector.js:39. Any benefits from that style? Or breaking 1000 LOC inspector.js is another pull request?

**Comment by nicknikolov** [comment]
New comment styles
screen shot 2018-03-21 at 11 09 16
